mxnet-predict-toolfile.spec

```diff
@@ -9,13 +9,14 @@ Requires: mxnet-predict
 mkdir -p %i/etc/scram.d
 cat << \EOF_TOOLFILE >%i/etc/scram.d/mxnet-predict.xml
 <tool name="mxnet-predict" version="@TOOL_VERSION@">
-  <lib name="mxnetpredict"/>
+  <lib name="mxnet"/>
   <client>
     <environment name="MXNET_PREDICT_BASE" default="@TOOL_ROOT@"/>
     <environment name="INCLUDE" default="$MXNET_PREDICT_BASE/include"/>
-    <environment name="LIBDIR" default="$MXNET_PREDICT_BASE/lib"/>
+    <environment name="LIBDIR" default="$MXNET_PREDICT_BASE/lib64"/>
   </client>
   <use name="openblas"/>
+  <use name="lapack"/>
```

Comment:
hi- why is this needed? better to use openblas instead.

Reply:
@davidlange6 The CMake build fails without providing the standalone LAPACK library. I put a workaround in https://github.com/cms-sw/cmsdist/pull/5148/commits/03a8093b04eba91a5dab1a80fccb726a3d411dfc.